plugins/pulp_rpm/plugins/importers/yum/depsolve.py

```diff
@@ -421,8 +432,6 @@ def get_requirements(self, units):
         :rtype:     generator
         """
         for segment in paginate(units):
-            unit_ids = [unit.id for unit in segment]
-            fields = ['requires', 'id']
-            for result in models.RPM.objects.filter(id__in=unit_ids).only(*fields):
-                for require in result.requires or []:
+            for unit in segment:
+                for require in unit.requires or []:
```

Comment:
This change looks a little complex to do so late in 2.8.0. What if we held it for 2.8.1 just in case something does break as a result? If anything downstream of this loaded these things in RAM, I'd be worried about it not limiting the fields like it used to. Also, according to the docblock this isn't being given Units, but the weirdo shadow Units in pulp.plugins (the model Unit is in pulp.server.something.models I think). So the models might not be equivalent, unless the docblock is wrong.